packages/bar/src/BarCanvas.tsx

```diff
@@ -367,6 +371,25 @@ const InnerBarCanvas = <RawDatum extends BarDatum>({
             }
         })
 
+        const barTotals = computeBarTotals(
+            enableTotals,
+            bars,
+            xScale,
+            yScale,
+            layout,
+            groupMode,
+            totalsOffset
+        )
+
+        ctx.fillStyle = theme.text.fill
+        ctx.font = `bold ${theme.text.fontSize}px sans-serif`
```

Comment:
I think you could use the following, which is used for similar components:

```
ctx.font = `${theme.labels.text.fontSize}px ${theme.labels.text.fontFamily}`
```

Reply:
I just implemented this way for svg and canvas